RileyLinkKit/UI/RileyLinkDeviceTableViewController.swift

```diff
@@ -474,6 +478,18 @@ public class RileyLinkDeviceTableViewController: UITableViewController, TextFiel
                     }
                     return NSLocalizedString("Fetching glucose…", comment: "Progress message for fetching pump glucose.")
                 }
+            case .writeGlucoseHistoryTimestamp:
+                vc = CommandResponseViewController { [unowned self] (completionHandler) -> String in
+                    self.device.ops?.writeGlucoseHistoryTimestamp() { (response) -> Void in
+                        switch response {
+                        case .success(_):
+                            completionHandler(String(describing: "Okay, thing worked"))
```

Comment:
`String(describing: "Okay, thing worked")` is more concisely written as `"Okay, thing worked"`. Future users might thank you if this was more descriptive...